packages/venia-ui/lib/components/TextInput/textInput.css

```diff
@@ -1,3 +1,6 @@
+.icons {
```

Comment:
I think this is leftover, right?

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2124/commits/67a1061ae9e205d3f676c3b035c5cd3048dcc8e0.